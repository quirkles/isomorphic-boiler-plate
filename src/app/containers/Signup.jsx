import React from 'react';

const Signup = () => {
  return (
    <div>
      <h3>Sign up</h3>
      <div>
        <form>
          <input type="text" name="email" placeholder="email" />
          <input type="password" name="password" placeholder="password" />
          <input type="text" name="confirm" placeholder="Confirm password" />
          <button>Sign up</button>
          <p>or</p>
          <button>Sign up with Facebook</button>
        </form>
      </div>
    </div>
  );
};

export default Signup;
