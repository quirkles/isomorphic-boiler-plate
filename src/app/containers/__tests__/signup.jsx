import React from 'react';
import { mount } from 'enzyme';

import Signup from '../Signup';

describe('Signup component', () => {
  it('Renders without errors and matches snapshot', () => {
    let wrapper;
    expect(() => {
      wrapper = mount(<Signup />);
    }).not.toThrow();
    expect(wrapper).toMatchSnapshot();
  });
});
