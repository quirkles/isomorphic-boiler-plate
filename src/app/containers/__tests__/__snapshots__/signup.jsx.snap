// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signup component Renders without errors and matches snapshot 1`] = `
<Signup>
  <div>
    <h3>
      Sign up
    </h3>
    <div>
      <form>
        <input
          name="email"
          placeholder="email"
          type="text"
        />
        <input
          name="password"
          placeholder="password"
          type="password"
        />
        <input
          name="confirm"
          placeholder="Confirm password"
          type="text"
        />
        <button>
          Sign up
        </button>
        <p>
          or
        </p>
        <button>
          Sign up with Facebook
        </button>
      </form>
    </div>
  </div>
</Signup>
`;
